### PR TITLE
runner/crusher drag speed nerf

### PR DESCRIPTION
**Pull request by 567Turtle**
# About the pull request

theyre quadrapeds, dragging things is hard

# Explain why it's good for the game

Runners were never really meant to be a capping caste and I think it's unintended that that's like 90% of their use right now. Currently Runners completely outclass Lurkers and I think this will even the playing field, with Lurkers being better for caps and runners being better for the skirmishing they're intended for.

Crushers were kinda tacked on to keep the logic consistent.

<details>
<summary>Screenshots & Videos</summary>

didnt test this but it prob works

</details>

# Changelog
:cl:
balance: Runners and Crushers now have a .5 movement speed modifier while dragging.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/Runner.dm [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
 /mob/living/carbon/xenomorph/runner/recalculate_actions()
 	. = ..()
-	pull_multiplier *= 0.85
+	pull_multiplier *= 0.5
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/castes/Runner.dm:83]: Don't do this here. Just change the value in the define for `/mob/living/carbon/xenomorph/runner`

File: code/modules/mob/living/carbon/xenomorph/castes/Crusher.dm [hunk 2]
```diff
@@ -39,6 +39,12 @@
 	tier = 3
 	drag_delay = 6 //pulling a big dead xeno is hard
 
+/mob/living/carbon/xenomorph/crusher/recalculate_actions()
+	. = ..()
+	pull_multiplier *= 0.5
+	if(is_zoomed)
+		zoom_out()
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/castes/Crusher.dm:46]: This is unnecessary.

File: code/modules/mob/living/carbon/xenomorph/castes/Crusher.dm [hunk 3]
```diff
@@ -39,6 +39,12 @@
 	tier = 3
 	drag_delay = 6 //pulling a big dead xeno is hard
 
+/mob/living/carbon/xenomorph/crusher/recalculate_actions()
+	. = ..()
+	pull_multiplier *= 0.5
```
Comment (Drulikar) [inline code/modules/mob/living/carbon/xenomorph/castes/Crusher.dm:44]: Don't do this here. Just change the value in the define for /mob/living/carbon/xenomorph/crusher

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Comment by ihatethisengine** [comment]
Lurker cannot solo cap at all, how this will make lurkers better at capping LOL. Have you even played the game? Runner is already extremely weak and dies to a single PB, why nerf the single utility base runner has? You just said it's 90% of base runner use and you just nerf it without a thought.

> theyre quadrapeds, dragging things is hard

https://github.com/cmss13-devs/cmss13/assets/115417687/4157e41a-9f94-4bf1-af2b-f3ae3e731fab

Cope?

**Comment by InsaneRed** [comment]
Runners are quite literally the designated capture caste.

Theye're a LOW HEALTH CASTE THAT CAN PICK THEIR FIGHTS. Like many classes throughout many games. There's a reason why the shove is limited to 4, because 4 shoves is usually how many shots you can take before being forced to retreat. Which means you're taking a risk to capture the person youre fighting against, because in most cases your slashes arent strong enough to be really doing any damage unless again you're one of the top players. Even in a one versus one a runner will have trouble capturing you unless they're quite literally the 1-3 people who play runner religiously, everyone else i've seen get shot and be forced to retreat or overestimate themselves and die. This smells PR smells of personal cope.

Lurkers are an ambush caste, they play similiar but differently from runners. If you're going to be making balance changes please provide actual reasons rather then "they're just quadruplets (you do realise theres quite literally an entire animal kingdom made of these things that drag their prey?)"

**Comment by RainbowStalin** [comment]
If you find it unrealistic that a quadruped drags their pray fast, then just say they use their mouth to drag instead of their hands (like most quadrupeds in the animal kingdom).

**Comment by Drulikar** [comment]
I can tell testing was not done w/ these values because these actually make dragging faster... If you legitimately wanted to find a good value, run it locally and just var edit the value to find a value you like. The existing values feel fine as is though for crusher and runner.

**Pull request closed without merging**